pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -62,25 +71,76 @@ def __init__(self, logger, handler, socket_path, platform_kind, namespace=None,
         # indicate that we're ready
         self._write_packet_to_processor('s')
 
+    @staticmethod
+    def _event_from_msgpack(parsed_data):
```

Comment:
please remove and make its callee to call `nuclio_sdk.Event.from_json` instead

Reply:
@erand asked me to do it in Nuclio itself and not the SDK.